src/main/kotlin/baseball/domain/ComputerBall.kt

```diff
@@ -0,0 +1,14 @@
+package baseball.domain
+
+private const val START_INDEX = 0
+
+class ComputerBall(val numbers: List<Int>) {
+
+    fun countContainingBalls(balls: List<Int>): Int {
+        return numbers.count { number -> balls.any { ball -> number == ball } }
```

Comment:
```suggestion
    return numbers.intersect(balls).size
```

intersect로 겹치는 element만 추출 할 수 있어요!